ITR/data/data_warehouse.py

```diff
@@ -37,6 +37,28 @@ def __init__(self, company_data: CompanyDataProvider,
         self.column_config = column_config
         self.company_data = company_data
         self.company_data._calculate_target_projections(benchmark_projected_production)
+        
+        # After projections have been made, shift S3 data into S1S2.  If we shift before we project,
```

Comment:
Is it a hack, to support automotive use-case?
I thought, our intention is to start supporting S3 properly, instead of shifting it into S1S2.

I might misunderstand the strategy, on the other hand

Reply:
See later comment about "producer-centric" benchmarks.  The code must follow the methodology.